app/test_main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+import pytest
+from unittest.mock import patch, MagicMock
+from app.main import can_access_google_page
+from freezegun import freeze_time
+
+
+@pytest.fixture
+def mock_valid_google_url() -> MagicMock:
+    with patch("app.main.requests.get") as mock_get:
+        yield mock_get
+
+
+def test_accessibility_with_valid_connection(
```

Comment:
You made three test which do exactly the same thing hence you can make one test and parametrize it.
Also you must have cover all combinations of possible responses of both mocked functions